### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.js -L  
technik,gir,nam,ans,wih,wil,ro,nowe,te,datas,qutie,ned,fo,clude,alph,crashers,nd,wasn,cros,  ue,possition`

### CODE REVIEW

File: doc/help/configuring.asciidoc [hunk 1]
```diff
@@ -120,7 +120,7 @@ c.completion.shrink = True
 ----
 
 Note that qutebrowser does some Python magic so it's able to warn you about
-mistyped config settings. As an example, if you do `c.tabs.possition = "left"`,
+mistyped config settings. As an example, if you do `c.tabs.position = "left"`,
```
Comment (The-Compiler) [inline doc/help/configuring.asciidoc:123]: This one is deliberate, see the context around it.

File: tests/end2end/data/hints/ace/ace.js [hunk 2]
```diff
@@ -8182,7 +8182,7 @@ function Folding() {
 
     this.foldAll = function(startRow, endRow, depth) {
         if (depth == undefined)
-            depth = 100000; // JSON.stringify doesn't hanle Infinity
+            depth = 100000; // JSON.stringify doesn't handle Infinity
```
Comment (The-Compiler) [inline tests/end2end/data/hints/ace/ace.js:8185]: This is in bundled third-party code (ace.js) - I'd prefer to keep it identical to the upstream copy.

**Review by The-Compiler — CHANGES_REQUESTED** [review]
Nice, thank you! I think two of them should be undone, but the rest looks great.

File: doc/changelog.asciidoc [hunk 3]
```diff
@@ -4423,7 +4423,7 @@ Changed
     * Add `-s`/`--space` argument to `:set-cmd-text` (as `:set-cmd-text "foo "` will now set the literal text `"foo "`)
 - Ignore `;;` for splitting with some commands like `:bind`.
 - Add unbound (new) default keybindings to config. This also adds a new `<unbound>` special command.
-    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be readded automatically.
+    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be read automatically.
```
Comment (mschilli87) [inline doc/changelog.asciidoc:4426]: I think this is another one that is wrong.
I'd suggest

```suggestion
    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be re-added automatically.
```

instead.

File: qutebrowser/mainwindow/mainwindow.py [hunk 4]
```diff
@@ -412,7 +412,7 @@ def _on_config_changed(self, option):
             self._set_decoration(config.val.window.hide_decoration)
 
     def _add_widgets(self):
-        """Add or readd all widgets to the VBox."""
+        """Add or re-add all widgets to the VBox."""
```
Comment (mschilli87) [inline qutebrowser/mainwindow/mainwindow.py:415]: Just highlighting this to show that my previous suggestion would be in-line with what was done here.

**Review by mschilli87 — CHANGES_REQUESTED** [review]
Just one minor think I think should be changed before merging this.

File: doc/changelog.asciidoc (hunk 3 above)
Comment (The-Compiler) [reply doc/changelog.asciidoc:4426]: Totally! That's what I get for reviewing this before my morning coffee, I thought _"how on earth did I manage to misspell 'read' this bad"_ even :rofl:

File: doc/help/configuring.asciidoc (hunk 1 above)
Comment (mschilli87) [reply doc/help/configuring.asciidoc:123]: This one still needs to be undone.

**Review by mschilli87 — CHANGES_REQUESTED** [review]
My finding was addressed, @The-Compiler's not (yet).

File: tests/end2end/data/hints/ace/ace.js (hunk 2 above)
Comment (mschilli87) [reply tests/end2end/data/hints/ace/ace.js:8185]: same here

Comment (mschilli87) [reply tests/end2end/data/hints/ace/ace.js:8185]: This is also still open.

File: qutebrowser/mainwindow/mainwindow.py (hunk 4 above)
Comment (mschilli87) [reply qutebrowser/mainwindow/mainwindow.py:415]: This is resolved but somehow Github does not allow me to mark it as such and/or hide this comment. :shrug:

**Review by mschilli87 — APPROVED** [review]
LGTM now :+1:

**Comment by The-Compiler** [comment]
Thanks @kianmeng and @mschilli87! I was kinda contemplating adding this as a linter to CI given its low noise, but it's probably more useful with a human in the loop. Guess we should just do this manually from time to time.

**Pull request merged**